### PR TITLE
Add interepretation writeup with plots

### PR DESCRIPTION
**Pull request by mfreed7**
Per the [request](https://github.com/web-platform-dx/developer-research/pull/22#issuecomment-1508382688), this PR adds an interpretation document with plots and tables of the results.

@dontcallmedom, LMK if this is roughly what you were looking for.

@foolip @robnyman thanks for the advice on this.

### CODE REVIEW

**Review by dontcallmedom — COMMENTED** [review]
Suggested improvements on the alternative text for the images of the charts, and a question on the content of one of the said charts - but overall, this LGTM

File: mdn-short-surveys/2023-03-31-web-components/interpretation.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+![Variation](variation.png)
+
+### Need vs. Don't Need
+
+![Need vs. Don't Need](need_vs_dont_need.png)
+
+### Split by Feature
+
+![Split by Feature](split_by_item.png)
```
[the remark this answers is not in the document]
Comment (mfreed7) [reply mdn-short-surveys/2023-03-31-web-components/interpretation.md]: This is a good point - I've updated the plot to correctly split by the response to the first question.

**Comment by mfreed7** [comment]
LMK if this is good to land (I don't have permission).

**Comment by dontcallmedom** [comment]
thanks a lot

**Pull request merged**